lldb/unittests/Language/CPlusPlus/CPlusPlusLanguageTest.cpp

```diff
@@ -397,3 +397,16 @@ TEST(CPlusPlusLanguage, CPlusPlusNameParser) {
   // Don't crash.
   CPlusPlusNameParser((const char *)nullptr);
 }
+
+TEST(CPlusPlusLanguage, DoesNotMatchCxx) {
```

Comment:
Since you're adding this here, can you add a case where we do match C++? Using Itanium and MS ABI examples?